client/app/lib/sidebar/components/ownedresourceslist.coffee

```diff
@@ -0,0 +1,288 @@
+debug = require('debug')('sidebar:ownedresourcelist')
+kd = require 'kd'
+cx = require 'classnames'
+React = require 'app/react'
+
+{ findDOMNode } = require 'react-dom'
+
+whoami = require 'app/util/whoami'
+isAdmin = require 'app/util/isAdmin'
+canCreateStacks = require 'app/util/canCreateStacks'
+isDefaultTeamStack = require 'app/util/isdefaultteamstack'
+
+List = require 'app/components/list'
+Link = require 'app/components/common/link'
+
+SidebarNoStacks = require 'app/components/sidebarstacksection/sidebarnostacks'
+
+SidebarMachineItem = require './machineitemcontainer'
+OwnedResourceHeader = require './ownedresourceheader'
+SidebarNoStacks = require './nostacks'
+
+MENU = null
+
+module.exports = class OwnedResourcesList extends React.Component
+
+  constructor: (props) ->
+    super props
+
+    @headers = []
+
+
+  onMenuItemClickError: (name) ->
+
+    new kd.NotificationView { title: "Error occured while #{name} template" }
```

Comment:
`Error occured while #{name}` **the** `template` also it would be better to use `show{Error, }Notification` util instead.